src/Components/CreateImageWizard/steps/ImageOutput/TargetEnvironment.tsx

```diff
@@ -13,6 +13,10 @@ import {
 } from '@patternfly/react-core';
 import { HelpIcon } from '@patternfly/react-icons';
 
+import awsIcon from '../../../../assets/aws.svg';
+import gcpIcon from '../../../../assets/google-cloud-short.svg';
+import azureIcon from '../../../../assets/microsoft-azure-short.svg';
+import oracleIcon from '../../../../assets/oracle-short.svg';
```

Comment:
I'm not sure if we can just ship these :grimacing:

Reply:
Ya I was a bit concerned about that. I'll leave them out for now. Also we're not doing any of the targets for the MVP. So should be fine.